mrjob/tools/emr/terminate_idle_clusters.py

```diff
@@ -51,8 +51,7 @@
                         this higher than the amount of time your jobs can take
                         to start instances and bootstrap.
   --max-mins-locked MAX_MINS_LOCKED
-                        Max number of minutes a cluster can be locked while
-                        idle.
+                        Deprecated, does nothing
```

Comment:
[Optional] Though, not really required but would it make sense to add [deprecation warning](https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings).